From 35edead362f6a4272ee7ad1482359292f12ce962 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Mon, 25 Nov 2019 07:57:25 -0600
Subject: [PATCH 25/34] ARM64: dts: Fix GPIO Expander compatible flag

The GPIO expanders on the baseboard are pcal6416A devices which
are software compatbile with the tca6416 when used as outputs, but
when configured as inputs, they are not.

This patch correct sets the compatible flag for the GPIO expanders
to "nxp,pcal6416"

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
index e64edfa815c1..6b1f4fe640e9 100644
--- a/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
@@ -339,6 +339,8 @@
 		reg = <0x21>;
 		gpio-controller;
 		#gpio-cells = <2>;
+		interrupt-parent = <&gpio4>;
+		interrupts = <27 IRQ_TYPE_LEVEL_LOW>;
 	};
 
 	wm8962: audio-codec@1a {
-- 
2.17.1

